packages/circuit-ui/components/Flag/Flag.tsx

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright 2025, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { forwardRef, type HTMLAttributes } from 'react';
+import { getIconURL, type IconName } from '@sumup-oss/icons';
```

Comment:
Importing `IconName` will require bumping the `@sumup-oss/icons` peer dependency to v5.16.0.